assets/section-footer.css

```diff
@@ -160,46 +155,94 @@
   }
 }
 
+.footer__follow-on-shop:empty,
 .footer__list-social:empty,
 .footer-block--newsletter:empty {
   display: none;
 }
 
+.footer__follow-on-shop {
+  display: flex;
+  text-align: center;
+}
+
 .footer__list-social.list-social:only-child {
   justify-content: center;
 }
 
 .footer-block__newsletter {
   text-align: center;
+  flex-grow: 1;
 }
 
 .newsletter-form__field-wrapper {
   max-width: 36rem;
 }
 
 @media screen and (min-width: 750px) {
+  /* Pushes other components to the right of the flexbox */
   .footer-block__newsletter:not(:only-child) {
     text-align: left;
+    margin-right: auto;
   }
 
   .footer-block__newsletter:not(:only-child) .footer__newsletter {
     justify-content: flex-start;
     margin: 0;
   }
 
-  .footer-block__newsletter:not(:only-child) .newsletter-form__message--success {
+  .footer-block__newsletter:not(:only-child)
+    .newsletter-form__message--success {
     left: auto;
   }
-}
 
-.footer-block__newsletter + .footer__list-social {
-  margin-top: 3rem;
+  .footer__follow-on-shop {
+    margin-bottom: 0.4rem;
+  }
+  /* Follow on shop is the first button but it has siblings*/
+  .footer__follow-on-shop:first-child:not(:last-child) {
+    justify-content: flex-start;
+    margin-right: auto;
+    text-align: left;
+  }
+
+  /*
+    All three components are present, email, Follow on Shop, and social icons.
+    Moves the FoS button next to the social icons so they appear grouped together
+  */
+  .footer__follow-on-shop:not(:first-child):not(:last-child) {
+    justify-content: flex-end;
+    text-align: right;
+  }
 }
 
 @media screen and (max-width: 749px) {
-  .footer__list-social.list-social {
+  /*
+    On a small screen we want all the items to be centered
+    because they will be stacked.
+   */
+  .footer-block--newsletter {
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 100%;
+    align-items: center;
+    gap: 3rem;
+  }
+
+  .footer__list-social.list-social,
+  .footer__follow-on-shop,
+  .footer-block__newsletter {
+    display: flex;
     justify-content: center;
   }
+
+  .footer-block__newsletter {
+    flex-direction: column;
+  }
+
+  .footer__follow-on-shop {
+    text-align: center;
+  }
```

Comment:
this isn't needed as it's on line `166` already 👍 
![](https://screenshot.click/13-11-37f77-m2x6v.png)